bundles/org.openhab.binding.nikohomecontrol/src/main/java/org/openhab/binding/nikohomecontrol/internal/protocol/nhc2/NikoHomeControlCommunication2.java

```diff
@@ -627,6 +631,9 @@ public void executeAction(String actionId, String value) {
                 } else if (NHCOFF.equals(value)) {
                     property.status = value;
                 } else {
+                    action.setState(Integer.parseInt(value)); // set cached state to new brightness value to avoid
```

Comment:
No need to catch exception?

Reply:
It would be a bug if the value is not an int here, but I will put in a catch and log it (and do it at a few more places in the same class when I am at it).